package.json

```diff
@@ -82,6 +82,7 @@
   ],
   "dependencies": {
     "glob": "~11.0.0",
+    "jiti": "2.4.2",
```

Comment:
Do we want jiti as a direct dependency? IMO we could either use it as peerDependency with peerDependencyMeta optional

Or we widen the range to ~